src/date.rs

```diff
@@ -35,7 +35,7 @@ impl Date {
     }
 
     /// Converts the `Date` to an XML plist date string.
-    pub fn to_xml_format(&self) -> String {
```

Comment:
It's going to be dropped anyway, because it's part of the second commit. But yes, stable API, good argument, even though it is just 16 bytes and it does implement `Copy`, which is why clippy does suggest changing that.e